task-random-item.js

```diff
@@ -0,0 +1,11 @@
+//3. Napisz funkcję randomItem(), która zwróci losowy element podanej tablicy.
+
+
+function randomItem(randomItem){
```

Comment:
The first parameter should be an array. The function should return a random item for the GIVEN array.